scripts/packager-codes/common/io.py

```diff
@@ -23,7 +23,41 @@
 import os
 import shutil
 
-PACKAGER_CODES_DIR = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'packager-codes')
+# Go up from common/ -> packager-codes/ -> scripts/ -> repo_root, then to packager-codes/
+PACKAGER_CODES_DIR = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))),
```

Comment:
nitpick: it's the type of expression where PathLib is great by providing a .parent attribute !